core/test/scenarios/__snapshots__/api-test-pptr.js.snap

```diff
@@ -409,7 +409,6 @@ Array [
   "image-alt",
   "image-aspect-ratio",
   "image-redundant-alt",
-  "image-size-responsive",
```

Comment:
I'm a bit concerned with this diff...
It seems `image-size-responsive` has been removed from the snapshot?
Is that expected?